frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -461,3 +573,126 @@ function deleteNodesWithNoDescendantsForTissueCardOntologyView(
     });
   }
 }
+
+function getInitialStateForSelectedGene(
```

Comment:
this first gets the list of valid cell types (the input cell types and all their ancestors).

then, it gets the initial tree state. this is the same implementation used to build the tissue initial ontology state in the cellguide pipeline ported over to typescript.